mkdocs.yml

```diff
@@ -43,11 +43,21 @@ markdown_extensions:
   - admonition
   - toc:
       permalink: true
-  # Enable the Snippets extension to add the ability to embed content from
-  # external files.
-  # Ref: https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#snippets
```

Comment:
Maybe we shouldn't remove a comment that serves as a rationale as to why the `pymdownx.snippets` extension is enabled? 😅 For future reference.